playbooks/tasks/portal-configs-load.yml

```diff
@@ -2,6 +2,7 @@
 # Get portal configs
 # - Load portal configs from secrets backend (e.g. LastPass)
 # - Set default values (for unset keys)
+# - Set mongodb mgkey (if defined but empty)
```

Comment:
```suggestion
# - Set mongodb mgkey (if either undefined or defined but empty)
```

Reply:
That is not exact:

It is like this:

`# - Set default values (for unset keys)`
- key is not set (= undefined) => the value is set here

`# - Set mongodb mgkey (if defined but empty)`
- key is set (= defined) but value is not set => the value is set here

Reply:
OK.